samples/metaapp/src/main/res/layout-television/property_list_title.xml

```diff
@@ -0,0 +1,50 @@
+<androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
merge the property_list_title.xml with property_list_title_leanback.xml in only one xml (https://github.com/SourcePointUSA/android-cmp-app/blob/develop/samples/metaapp/src/main/res/layout-television/property_item.xml)

Reply:
replace 
`androidx.constraintlayout.widget.ConstraintLayout`
with
`com.sourcepointmeta.metaapp.tv.properties.PropertyListTitleView`